iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/connector/protocol/IoTDBConnector.java

```diff
@@ -125,4 +138,39 @@ protected Set<TEndPoint> parseNodeUrls(PipeParameters parameters) {
 
     return givenNodeUrls;
   }
+
+  /**
```

Comment:
I deem it better to do another abstract upon the "IoTDBConnector", and make all the connectors need commiting extend it. Therefore we can avoid duplicateing the commit logic.

Reply:
i think this is good